man/src/plugin/ls.md

```diff
@@ -0,0 +1,43 @@
+Lists all the plugins that are currently installed. You can install plugins
+using the `docker plugin install` command.
+You can also filter using the `-f` or `--filter` flag.
+
+## Filters
+
+Filter output based on these conditions:
+   - enabled=(true|false) - plugins that are enabled or not
+   - capability=<string> - filters plugins based on capabilities (currently `volumedriver`, `networkdriver`, `ipamdriver`, or `authz`)
+
+## Format
+
+   Pretty-print plugins using a Go template.
+   Valid placeholders:
+      .ID - Plugin ID.
+      .Name - Plugin Name.
+      .Description - Plugin description.
+      .Enabled - Whether plugin is enabled or not.
+
+# EXAMPLES
+# Display all plugins
+
+    # docker plugin ls
```

Comment:
@thaJeztah Done. Also another PR has been #30660 created to cover other markdown files. Please take a look.